services/hanke-service/src/main/resources/db/changelog/changesets/024-connect-application-to-hanke.yml

```diff
@@ -0,0 +1,16 @@
+databaseChangeLog:
+  - changeSet:
+      id: 024-connect-application-to-hanke
+      author: Niko Junkala
+      changes:
+        - addColumn:
+            tableName: applications
+            columns:
+              - column:
+                  name: hanke_id
+                  remarks: "Add relating hanke to application"
+                  type: bigint
+                  constraints:
+                    foreignKeyName: fk_application_hanke
+                    references: hanke(id)
+                    nullable: false
```

Comment:
Empty line should be added to end of file.

Reply:
Fixed file settings of the IDE.